ctx.go

```diff
@@ -479,6 +476,38 @@ func (ctx *Ctx) Fresh() bool {
 	return true
 }
 
+// isNoCache checks if the cacheControl header value is a `no-cache`.
+func isNoCache(cacheControl string) bool {
```

Comment:
to be clear, this function is implemented to replicate the previous regexp `(?:^|,)\s*?no-cache\s*?(?:,|$)`. If we think having random chars before `,no-cache,` is invalid too, happy to change it too.

Reply:
LGTM